stack.yaml

```diff
@@ -64,6 +64,11 @@ packages:
     git: https://github.com/input-output-hk/cardano-crypto
     commit: 1cde8e3a8d9093bbf571085920045c05edb3eaa4
   extra-dep: true
+- location:
+    git: https://github.com/serokell/log-warper
+    commit: a2db552a9ce0fd58d82e3d1ffd147a8474d8db5c # master
+  extra-dep: true
```

Comment:
Let's use Hackage version, I am going to upload it shortly.
And CI failed because you didn't update default.nix.

Reply:
https://github.com/serokell/log-warper/pull/30

Reply:
https://hackage.haskell.org/package/log-warper